flightcontrol/flightcontrol.json

```
@@ -15,8 +15,8 @@
           "cpu": 0.25,
           "memory": 0.5,
           "port": 8911,
-          "installCommand": "cd flightcontrol && NODE_ENV=development yarn install --immutable",
-          "buildCommand": "cd flightcontrol && yarn rw deploy flightcontrol",
+          "installCommand": "yarn set version berry && cd flightcontrol && NODE_ENV=development yarn install --immutable",
+          "buildCommand": "cd flightcontrol && yarn rw deploy flightcontrol api",
```

```suggestion
          "installCommand": "yarn set version stable && cd flightcontrol && NODE_ENV=development yarn install --immutable",
          "buildCommand": "cd flightcontrol && yarn rw deploy flightcontrol api",
```

They're the same thing, but we just prefer `stable` over `berry`.

Oof, missing `api`... sorry about that 🤦‍♂️